docs/application-development/foundations.md

```diff
@@ -322,269 +326,259 @@ values={[{label: 'Go', value: 'go'},{label: 'Java', value: 'java'},{label: 'PHP'
 
 <TabItem value="go">
 
-In the Temporal Go SDK programming model, a [Workflow Definition](/workflows#workflow-definitions) is an exportable function.
+Use the [`NewClient()`](https://pkg.go.dev/go.temporal.io/sdk@v1.8.0/client#NewClient) API available in the [`go.temporal.io/sdk/client`](https://pkg.go.dev/go.temporal.io/sdk@v1.8.0/client) package to create a new [`Client`](https://pkg.go.dev/go.temporal.io/sdk@v1.8.0/client#Client)
```

Comment:
`NewClient` is deprecated, prefer `Dial`. Also, no Go API doc links should have `@v<version>`. Meaning, https://pkg.go.dev/go.temporal.io/sdk@v1.8.0/client is bad, https://pkg.go.dev/go.temporal.io/sdk/client is proper.